test/e2etest/installer_test.go

```diff
@@ -32,6 +35,11 @@ func TestInstaller(t *testing.T) {
 	if contains(productList, confluence) {
```

Comment:
```suggestion
	println(fmt.Sprintf("################## Confluence Tests ##################", testConfig.AwsRegion))
	if contains(productList, confluence) {
```

Reply:
Actually it needs a `%s` if we want to include `AwsRegion`, but in this case I think we can simple have:
```
println("################## Confluence Tests ##################")
```